build.gradle

```diff
@@ -422,6 +420,15 @@ subprojects {
         "-Xlint:unused"
       ]
     }
+    
+// these options are valid for Scala versions < 2.13 only
+// Scala 2.13 removes them, see https://github.com/scala/scala/pull/6502 and https://github.com/scala/scala/pull/5969 
```

Comment:
Nit: can we indent these comments properly please?

Reply:
Done